cl/alerts/management/commands/cl_send_alerts.py

```diff
@@ -115,6 +127,9 @@ def __init__(self, *args, **kwargs):
         }
         self.options = {}
         self.valid_ids = {}
+        self.o_es_alerts = False
+        if waffle.switch_is_active("o-es-alerts-active"):
+            self.o_es_alerts = True
```

Comment:
I think we can use a ternary if statement:

```python
self.o_es_alerts = (
    True if waffle.switch_is_active("o-es-alerts-active") else False
)
```

Reply:
Yeah, changed!

Reply:
or just `self.o_es_alerts = waffle.switch_is_active("o-es-alerts-active")`